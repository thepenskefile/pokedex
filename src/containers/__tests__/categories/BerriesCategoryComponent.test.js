import React from 'react';
import BerriesCategoryComponent from '../../categories/BerriesCategoryContainer';
import renderer from 'react-test-renderer';

const defaultProps = {
  id: 'mockId'
};

it('renders correctly', () => {
  const component = renderer.create(<BerriesCategoryComponent {...defaultProps} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
