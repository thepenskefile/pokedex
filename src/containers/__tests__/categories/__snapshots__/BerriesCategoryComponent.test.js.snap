// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Box-sc-6sssmg-0 inpIDs"
  style={
    Object {
      "textAlign": "center",
    }
  }
/>
`;
